src/Symfony/Bridge/Doctrine/Tests/Security/User/EntityUserProviderTest.php

```diff
@@ -125,7 +125,7 @@ public function testLoadUserByUserNameShouldDeclineInvalidInterface()
     private function getManager($em, $name = null)
     {
         $manager = $this->getMock('Doctrine\Common\Persistence\ManagerRegistry');
-        $manager->expects($this->once())
+        $manager->expects($this->any())
```

Comment:
We shouldn't really care how many times `getManager()` is called anyways. And now, it's called multiple times in some tests, and zero times in others.